app/models/suggested_councillor.rb

```diff
@@ -2,4 +2,6 @@ class SuggestedCouncillor < ActiveRecord::Base
   has_one :authority, through: :councillor_contribution
   belongs_to :councillor_contribution
   validates :councillor_contribution, :name, :email, presence: true
+  validates_email_format_of :email, :message => "format seems not right.
```

Comment:
This should be in the newer Ruby has style `message: "value"`, not `:message => "value"`.

RuboCop should be picking this up. Are you running that before committing @hisayohorie ?